noggin/templates/user-settings-otp.html

```diff
@@ -114,6 +114,9 @@ <h5 id="pageheading">{{ _("OTP Tokens") }}</h5>
         $('#otp-toggle').click(function() {
           $('#otp-qrcode').slideToggle("fast");
         })
+        $('#otp-modal').on("hide.bs.modal", function (e) {
+          if(!confirm("{{_('Are you sure you want to close the OTP dialog? This code cannot be shown again, and you will be locked out of your account if you have no way to generate your an OTP')}}")) return false;
```

Comment:
> and you will be locked out of your account if you have no way to generate your an OTP

I'm not parsing the end of that sentence right.

Reply:
thanks! wording updated. It's still meh, but at least parses. :)